### PR TITLE
Add debug output to ovextcap

### PR DESCRIPTION
**Pull request by desowin**
This makes it possible to create both SOF and/or NAK filtered and
unfiltered pcap files at the same time.

### CODE REVIEW

**Comment by matwey** [comment]
Could you please manage CI failures?

**Comment by desowin** [comment]
> Could you please manage CI failures?

Fixed.

**Comment by desowin** [comment]
Apparently extcap fileselect in Wireshark does not really implement the mustexist property (it does only seem to affect the actual check for file before starting extcap, but does not change the file selection dialog at all). I will check it out and in the meanwhile I have removed it from ovextcap.

See https://code.wireshark.org/review/35643 for the Wireshark change.

File: tools/ovextcap/main.c [hunk 1]
```diff
@@ -378,26 +400,35 @@ static void print_extcap_dlts(void) {
...
+	       "{type=boolflag}{default=false}{group=Capture}\n");
 	printf("arg {number=2}{call=--filter-sof}"
 	       "{display=Filter Full and High speed Start-of-Frame packets}"
 	       "{tooltip=Only SOFs that do not interrupt any transaction will be filtered}"
-	       "{type=boolflag}{default=false}\n");
+	       "{type=boolflag}{default=false}{group=Capture}\n");
+
+	printf("arg {number=800}{call=--overwrite-debug-pcap}{display=Overwrite .pcap file is exists}{type=boolflag}"
+	       "{required=false}{default=false}{group=Debug}\n");
+	printf("arg {number=801}{call=--debug-pcap}{display=Save all packets to .pcap file}{type=fileselect}"
```
Comment (desowin) [inline tools/ovextcap/main.c]: It looks like there is fileext option available that could be used here.

File: tools/ovextcap/main.c [hunk 2]
```diff
@@ -378,26 +400,35 @@ static void print_extcap_dlts(void) {
...
 	       "{tooltip=NAKed SPLIT transactions won't be filtered}"
-	       "{type=boolflag}{default=false}\n");
+	       "{type=boolflag}{default=false}{group=Capture}\n");
 	printf("arg {number=2}{call=--filter-sof}"
 	       "{display=Filter Full and High speed Start-of-Frame packets}"
 	       "{tooltip=Only SOFs that do not interrupt any transaction will be filtered}"
-	       "{type=boolflag}{default=false}\n");
+	       "{type=boolflag}{default=false}{group=Capture}\n");
+
+	printf("arg {number=800}{call=--overwrite-debug-pcap}{display=Overwrite .pcap file is exists}{type=boolflag}"
```
Comment (desowin) [inline tools/ovextcap/main.c]:
```suggestion
	printf("arg {number=800}{call=--overwrite-debug-pcap}{display=Overwrite .pcap file if it exists}{type=boolflag}"
```

**Comment by matwey** [comment]
@desowin so, is it ready to be merged?

**Comment by desowin** [comment]
> @desowin so, is it ready to be merged?

I would prefer to wait for the [Wireshark change](https://code.wireshark.org/review/35643) to be reviewed first. I'll then fix the typo I noted above (assuming no changes to the actual extcap config are needed) and let you know that it's ready.

**Comment by matwey** [comment]
Ok, I see.

**Comment by desowin** [comment]
The Wireshark change has been merged without modifications and thus this is now ready for merge.

**Pull request merged**